Start_applications.bat

```diff
@@ -7,6 +7,8 @@ if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
+set ENV_NAME=geoapps
+
 set MY_CONDA=!MY_CONDA_EXE:"=!
-call "!MY_CONDA!" activate geoapps && jupyter notebook geoapps\index.ipynb
-cmd /k
+call "!MY_CONDA!" run -n %ENV_NAME% jupyter notebook geoapps\index.ipynb
+cmd /k "!MY_CONDA!" activate %ENV_NAME%
```

Comment:
Same comment as on GEOPY-891, kind of weird that you activate after.

Reply:
Does that work?

Reply:
yes it does work. Actually, if jupyter starts, it never reaches that point.
If juyter fails, then it places the user in a terminal where the environment is activated. Possibly useful for diagnostic

Reply:
I see. Great.